html-block-and-inline/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Html-Block-Inline</title>
+  </head>
+  <body>
+    <div>
+      <h1><b>Blue Americana Snuggie&reg; Blanket</b></h1>
```

Comment:
in this case, you do not need to add a `b` tag for bold text, as the text in a `h` element is bold by default!